lib/internal/cli_table.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const ansiRegex = () =>
+  new RegExp([
+    '[\\u001B\\u009B][[\\]()#;?]*(?:(?:(?:[a-zA-Z\\d]*',
+    '(?:;[a-zA-Z\\d]*)*)?\\u0007)',
+    '(?:(?:\\d{1,4}(?:;\\d{0,4})*)?[\\dA-PRZcf-ntqry=><~]))',
+  ].join('|'), 'g');
+
+const { Buffer } = require('buffer');
+
+const TableChars = {
+  /* eslint-disable node-core/non-ascii-character */
+  middle_middle: '─',
+  row_middle: '┼',
+  top_right: '┐',
+  top_left: '┌',
+  left_middle: '├',
+  top_middle: '┬',
+  bottom_right: '┘',
+  bottom_left: '└',
+  bottom_middle: '┴',
+  right_middle: '┤',
+  left: '│ ',
+  right: ' │',
+  middle: ' │ ',
+  /* eslint-enable node-core/non-ascii-character */
+};
+
+function countSymbols(string) {
+  const normalized = string.replace(ansiRegex(), '').normalize('NFC');
+  return Buffer.from(normalized, 'UCS-2').byteLength / 2;
+}
+
+const renderRow = (row, columnWidths) => {
+  let out = TableChars.left;
+  for (var i = 0; i < row.length; i++) {
+    const cell = row[i];
+    const len = countSymbols(cell);
+    const needed = (columnWidths[i] - len) / 2;
+    out += `${' '.repeat(needed)}${cell}${' '.repeat(Math.ceil(needed))}`;
```

Comment:
This is inconsistent: please move the `Math.ceil` call to the upper line so `needed` is already the ceil value. Otherwise the first repeat might be called with a floating point number. Or is that intended behavior? In that case a comment would be good.

Reply:
one is ceil and the other is normal rounding, so that when we have floating results we end up always pushing the element to the leftmost center instead of rightmost center

Reply:
Can you please still add a comment here?